src/controller.py

```diff
@@ -151,26 +212,45 @@ def run_command_and_get_its_response(self) -> dict:
         elif self.command == "emulator-reset-device":
             emulator.reset_device()
             return {"response": "Device reset"}
-        elif self.command == "bridge-start":
-            version = self.request_dict.get("version") or binaries.BRIDGES[0]
-            bridge.start(version, proxy=BRIDGE_PROXY)
-            response_text = f"Bridge version {version} started"
-            if BRIDGE_PROXY:
-                response_text = response_text + " with bridge proxy"
-            return {"response": response_text}
-        elif self.command == "bridge-stop":
-            bridge.stop(proxy=BRIDGE_PROXY)
-            response_text = "Stopping bridge"
-            if BRIDGE_PROXY:
-                response_text = response_text + " + stopping bridge proxy"
-            return {"response": response_text}
-        elif self.command == "exit":
-            emulator.stop()
-            bridge.stop()
-            log("Exiting", "red")
-            exit(1)
         else:
-            return {"success": False, "error": f"Unknown command - {self.command}"}
+            return {
+                "success": False,
+                "error": f"Unknown emulator command - {self.command}",
+            }
+
+    def run_regtest_command(self) -> dict:
+        if self.command == "regtest-mine-blocks":
+            block_amount = self.request_dict["block_amount"]
+            address = self.request_dict.get("address") or REGTEST_RPC.getnewaddress()
+            REGTEST_RPC.generatetoaddress(block_amount, address)
+            return {"response": f"Mined {block_amount} blocks by address {address}"}
+        elif self.command == "regtest-fill-address":
+            # NOTE: this is sending specified amount to address, if we wanted
+            #   to enforce the address has exactly X BTC, it would be harder
+            btc_amount = self.request_dict["btc_amount"]
+            address = self.request_dict["address"]
+
+            # Making sure we got enough BTC to send
+            # (including fees and considering a big 1kvB transaction)
+            while True:
+                our_balance = REGTEST_RPC.getbalance()
+                if our_balance > btc_amount + FEES_PER_KVB:
+                    break
+                else:
+                    REGTEST_RPC.generatetoaddress(1, REGTEST_RPC.getnewaddress())
+                    log(
+                        f"Our balance {our_balance} is smaller than {btc_amount}. Mining a new block to increase it."
+                    )
```

Comment:
This should be taken care of outside of tenv as well IMHO. @vdovhanych I think the container should be in a state where it is ready to be used directly. That means with a wallet created and with lot's of BTC.

Reply:
It should be possible to add it in the container from the startup. Will try to add it in the [pr.](https://github.com/trezor/trezor-user-env/pull/98)